crystal-map-connector/src/test/kotlin/ch/ergon/dope/extensions/aggregator/ArrayAggTest.kt

```diff
@@ -10,10 +11,18 @@ import ch.ergon.dope.helper.someCMStringList
 import ch.ergon.dope.resolvable.expression.unaliased.aggregator.AggregateQuantifier.ALL
 import ch.ergon.dope.resolvable.expression.unaliased.aggregator.ArrayAggregateExpression
 import ch.ergon.dope.toDopeType
+import kotlin.test.BeforeTest
 import kotlin.test.Test
 import kotlin.test.assertEquals
 
 class ArrayAggTest {
+    private lateinit var manager: DopeQueryManager
```

Comment:
Maybe also add a `ManagerDependentTest` that has the `setUp` function

Reply:
done